config/database.yml

```diff
@@ -81,5 +82,5 @@ test:
 production:
   <<: *default
   database: my_crud_production
-  username: my_crud
-  password: <%= ENV['MY_CRUD_DATABASE_PASSWORD'] %>
+  username: Diego
```

Comment:
This probably will cost us some time.
Just change the database name, let the default values for the rest of params